scripts/rollup/build.js

```diff
@@ -473,7 +473,7 @@ function handleRollupError(error) {
       highlightCode: true,
     });
     console.error(frame);
-  } else {
+  } else if (error.codeFrame) {
```

Comment:
Should we add an `else` here with something (e.g. "Unknown error")?

Reply:
We already print the error message itself earlier so these are additive.